nsqd/protocol_v2.go

```diff
@@ -399,7 +399,7 @@ func (p *protocolV2) IDENTIFY(client *clientV2, params [][]byte) ([]byte, error)
 		if identifyData.DeflateLevel <= 0 {
 			deflateLevel = 6
```

Comment:
one more small thing - setting `deflateLevel` here has no effect, I think we want to set `identifyData.DeflateLevel` to the default of 6 here